Modules/Vector/Vector2D.py

```diff
@@ -1,17 +1,24 @@
 #/usr/bin/python3
 from math import sqrt
+import math
 
 class vector2D:
 
     def __init__(self, x : float = 0, y : float = 0):
         self.x = x
         self.y = y
-
-    def distance_to(self, vector : vector2D) -> float:
+    def distance_to(self, vector:vector2D) -> float:
```

Comment:
Is it considered fixing the problem or uglifying the line?

SPACES.

And add quotes around `vector2D` (e.g. `"vector2D"`) so Python does forward references (prevents the `NameError`)